src/python/pants/backend/python/util_rules/interpreter_constraints_test.py

```diff
@@ -531,3 +533,13 @@ def test_major_minor_version_when_single_and_entire(
         assert set(all_versions) == {
             (*expected, patch) for patch in range(_PATCH_VERSION_UPPER_BOUND + 1)
         }
+
+
+def test_parse_python_interpreter_constraint() -> None:
```

Comment:
If we keep the `if "CPython" in constraint:` codepath, could we add a test?

Reply:
The check has been removed, so not relevant any longer.